test/validation/api/ipsec/ipsec_test_out.c

```diff
@@ -1284,6 +1284,7 @@ static void test_sa_info(void)
 
 	param_in.inbound.antireplay_ws = 32;
 	sa_in = odp_ipsec_sa_create(&param_in);
+	CU_ASSERT_FATAL(sa_in != ODP_IPSEC_SA_INVALID);
```

Comment:
CU_ASSERT_NOT_EQUAL_FATAL() was used for sa_out case. Do you prefer using only CU_ASSERT()/CU_ASSERT_FATAL()? I think it would be better to keep the usage uniform.

Reply:
I prefer CU_ASSERT as I feel it results in more readable code and narrower CUnit API usage.

It seems that CU_ASSERT is used more often in validation tests than CU_ASSERT_EQUAL and friends. IPsec validation tests use the latter more often although CU_ASSERT is used there also. I do not feel that consistency would require the newly added code to use CU_ASSERT_NOT_EQUAL_FATAL here. I can do the change,, but I would prefer not to.

Reply:
To me CU_ASSERT_FATAL() are easier to read than e.g. CU_ASSERT_NOT_EQUAL_FATAL(). I think we should replace usage of CU_ASSERT_EQUAL, _NOT_EQUAL, etc CU macros with CU_ASSERT and CU_ASSERT_FATAL macros.

Reply:
Agreed that the readability is better with CU_ASSERT and CU_ASSERT_FATAL. I am fine with the usage in this PR.